.github/workflows/build.yaml

```diff
@@ -1,11 +1,9 @@
 ---
 name: build
 on:
-  push:
+  pull_request:
     branches:
-      - main
-    tags:
-      - v*.*.*
+      - '**'
```

Comment:
Let's do both push and pull request events:

```yaml
on:
  push:
    branches:
      - '**'
  pull_request:
    - opened
    - edited
```

This ensures that pushes get tested, so that we quickly get feedback when something is broken.

Reply:
This is added.